packages/react-router-native/jest.config.js

```diff
@@ -8,9 +8,7 @@ module.exports = {
   globals: {
     __DEV__: true,
   },
-  modulePaths: [
-    "<rootDir>/node_modules", // for react-native
-  ],
+  transformIgnorePatterns: [],
```

Comment:
The `modulePaths` option as configured didn't work with pnpm's folder structure. A simpler way to solve the same problem is to simply tell Jest to transform everything.